src/index.js

```diff
@@ -256,8 +256,7 @@ class DropdownTreeSelect extends Component {
       }
       return
     } else if (e.key === 'Backspace' && tags.length && this.searchInput.value.length === 0) {
-      const lastTag = tags.pop()
-      this.onCheckboxChange(lastTag._id, false)
+      this.onTagRemove(tags[tags.length - 1]._id, true)
```

Comment:
@ellinge Do you remember why it was done like this? 

The problem with previous approach was that `onTagRemove` was never fired if you remove tags with `backspace` key. And thus `getNextFocusAfterTagDelete` was also not being invoked.

Reply:
Propably because the focus should be kept on the search field after tag removal?
We are checking if they/you are using backspace in an empty search field and there are tags left. In this case remove the last one (and keep focus on the search field). What is the result of getNext in this case? The search field?

Reply:
Hmm, after removing the last tag the focus goes back to input. I’d need to double check but I’m that’s the case as it would have stood out. 

So if we wanna keep focus on the search field after tag removal, this code (the keybordNavigation.focus... handler will be unreachable code as no one else invokes it).

Reply:
Here's how it looks on different browser with this change:

**Edge** (the focus is not as pronounced, cursor stays in the input (but input is not in focus - if you type anything it doesn't regisetr)
![xi7rTgSoQU](https://user-images.githubusercontent.com/781818/72747282-2a519b80-3b82-11ea-8ed2-a8cb81a1a828.gif)

Chrome (focus is pronounced, cursor doesn't stay in input until you clear all tags)
![8F0ZfZoAA1](https://user-images.githubusercontent.com/781818/72747292-3178a980-3b82-11ea-8ec6-34c52480055f.gif)

Reply:
This is the current implementation. The idea is that the focus is kept on the input field after using backspace. Do you know any WCAG-guidelines stating it should behave differently?

I also checked with react-select, it's how they have done as well. Not stating they are the answer.


- Edge

![focusedge](https://user-images.githubusercontent.com/17863113/73616092-5acc1780-460f-11ea-9606-2f58d07bffa7.gif)

- Chrome

![focuschrome](https://user-images.githubusercontent.com/17863113/73616093-5b64ae00-460f-11ea-9474-e57a6404b728.gif)

- Firefox

![focusfirefox](https://user-images.githubusercontent.com/17863113/73616094-5b64ae00-460f-11ea-944d-47b1c83f0b55.gif)

Reply:
Honestly, I haven't looked into WCAG as I trust your research on it. I've no objections either way. The only question, if we keep current implementation is this:

> So if we wanna keep focus on the search field after tag removal, this code (the keybordNavigation.focus... handler will be unreachable code as no one else invokes it).

I just wanna make sure I'm not missing anything and if this is dead code, then we should take it out. Thoughts?

Reply:
![getNextFocusAfterTagDelete](https://user-images.githubusercontent.com/17863113/75391181-e0c73f80-58e9-11ea-97d7-27a03e9bfe6b.gif)

It get's called when tabbing on a specific tag and use backspace